src/main/java/com/google/cloud/teleport/templates/PubSubToSplunk.java

```diff
@@ -363,4 +364,9 @@ public void processElement(ProcessContext context) {
                   }));
     }
   }
+
```

Comment:
Can you add a JavaDoc here to clarify what we are doing with this static method?

Reply:
Done